katana/katana.py

```diff
@@ -90,6 +93,207 @@ def readconfig():
     cfg = json.loads("".join(lines))
     return cfg
 
+@route('/searchkw', method='POST')
+def searchkw():
+    """
+    This method returns the ActionFile Path of the selected Driver.
+    """
+    value = parseString("".join(request.body))
+    indented_xml = "".join(value.toprettyxml(newl='\n'))
+    corrected_xml = remove_extra_newlines_char_xml(indented_xml)
+    corrected_xml = corrected_xml
+    with open('obj.txt', 'w') as f:
+        f.write(corrected_xml)
+    tree = ET.parse('obj.txt')
```

Comment:
What's the purpose of parsing the xml, write to a tmp file and re-read it again?
Is is possible to parse it once with the correct format? Or have the formatting done in memory and not in file.

Reply:
addressed